index.html

```diff
@@ -0,0 +1,104 @@
+        </ul>
+      </div>
+    </nav>
+    <div class="intro">
+      <span>"Hello! Young Techie"</span>
+      <h1 id="hero-text">
+        Project Managers <br> Global Workshop <br> 2022
+      </h1>
+    </div>
+    <p id="keynote">
+      Novustack is beginning the second quarter of 2022 with a fantastic offer for young techies looking to get their 
+      foot in the door of Product Management. 
+    </p>
+    <p class="event-date">2022.04.19(TUE) ~ 21(THU)</p>
+    <p class="event-venue">@ 50 Ebitu Ukiwe St, Jabi 900108, Abuja, Nigeria</p>
+  </header>
+  <main>
+    <section id="main-program">
+
+      <h2>Main Program</h2>
+      <hr class="heading-underline">
+      <div class="program-cards-container">
+        <div class="program-card">
+          <img src="./assets/img/lecture-icon.png" alt="Lecture Icon">
+          <h2 class="card-title">Lecture</h2>
+          <p>Listen to, among other speakers, a Harvard and MIT graduate with expertise as a former 
+            Amazon Technical Product Manager</p>
+        </div>
+        <div class="program-card">
+          <img src="./assets/img/exhition-icon.png" alt="Exhibition Icon">
+          <h2 class="card-title">PM Displays</h2>
+          <p>Appreciate various products made by proper application of project management concepts</p>
+        </div>
+        <div class="program-card">
+          <img src="./assets/img/forum-icon.png" alt="Forum Icon">
+          <h2 class="card-title">Forum</h2>
+          <p>Have time to discuss your ideas and thoughts with specialists on each subject.</p>
+        </div>
+        <div class="program-card">
+          <img src="./assets/img/workshop-icon.png" alt="Workshop Icon">
+          <h2 class="card-title">Workshop</h2>
+          <p>Try creating your own using open source license rather than just watching</p>
+        </div>
+        <div class="program-card">
+          <img src="./assets/img/network-icon.png" alt="Network Icon">
+          <h2 class="card-title">PM Ignite</h2>
+          <p>Get opportunies to network with Novustack affiliate around the world, also after the summit</p>
+        </div>
+      </div>
+      <button type="button">Join PM Global Workshop 2022</button>
+    </section>
+
+    <section id="speakers"></section>
+
+  </main>
+  <section class="partners">
+    <h2>Partners</h2>
+    <hr class="heading-underline">
+    <div class="logo-images-cont">
+      <img src="./assets/img/mozilla-logo.png" alt="Mozilla">
+      <img src="./assets/img/google-logo.png" alt="Google">
+      <img src="./assets/img/naver-Logo.png" alt="Naver Corporation">
+      <img src="./assets/img/daumkakao_icon.png" alt="Daumkakao">
+      <img src="./assets/img/airbnb-logo.png" alt="Air BNB">
+      <div class="partners-overlay"></div>
+    </div>
+  </section>
+  <footer>
+    <p class="copy">&copy;2022 Novustack.</p>
+    <p class="copyright">All rights reserved</p>
+  </footer>
```

Comment:
- [ ] The content of your footer is centered. Please align the logo to the left, and the text to the right as shown below 👇
### Your Design
![image](https://user-images.githubusercontent.com/46839548/164912258-1c408ce8-3463-4d71-97ce-02c577c8b73c.png)
### Behance Design
![image](https://user-images.githubusercontent.com/46839548/164912283-366c695e-eade-47bd-802f-7709774fe70e.png)